packages/plans-grid-next/src/components/features-grid/plan-features-list.tsx

```diff
@@ -65,14 +65,18 @@ const PlanFeaturesList = ( {
 			);
 
 			if ( featureGroup && ! filteredWpcomFeatures.length ) {
-				// Render a placeholder to keep the grid aligned
-				return (
-					<PlanDivOrTdContainer
-						key={ `${ planSlug }-${ mapIndex }` }
-						isTableCell={ options?.isTableCell }
-						className="plan-features-2023-grid__table-item"
-					/>
-				);
+				if ( options?.isTableCell ) {
+					// Render a placeholder to keep the grid aligned
+					return (
+						<PlanDivOrTdContainer
+							key={ `${ planSlug }-${ mapIndex }` }
+							isTableCell={ options?.isTableCell }
```

Comment:
Just out of interest, what does the change here (the added `if` condition) do in terms of the layout?

Also it looks like `<PlanDivOrTdContainer isTableCell ...` might suffice, due to the existing `options?.isTableCell` truthy check.

Reply:
> Just out of interest, what does the change here (the added if condition) do in terms of the layout?

On desktop, this adds placeholders so that the grid is aligned when a plan does not have any features for a particular feature group. 

<img width="1338" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5436027/f5745028-c224-4025-b9d8-0fed71ec1a80">

> Also it looks like <PlanDivOrTdContainer isTableCell ... might suffice, due to the existing options?.isTableCell truthy check.

Nice catch! I'll fix this.